lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -59,19 +59,10 @@ function runA11yChecks() {
     // @ts-expect-error
     const augmentAxeNodes = result => {
       // @ts-expect-error
-      result.nodes.forEach(node => {
-        // @ts-expect-error - getNodePath put into scope via stringification
-        node.path = getNodePath(node.element);
-        // @ts-expect-error - getOuterHTMLSnippet put into scope via stringification
-        node.snippet = getOuterHTMLSnippet(node.element);
-        // @ts-expect-error - getBoundingClientRect put into scope via stringification
-        const rect = getBoundingClientRect(node.element);
-        if (rect.width > 0 && rect.height > 0) {
```

Comment:
does the removal of this `if` end up affecting any audit results?

Reply:
It should be fine, I didn't need to add that guard.